src/main/java/org/opentripplanner/middleware/otp/response/Itinerary.java

```diff
@@ -0,0 +1,96 @@
+package org.opentripplanner.middleware.otp.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * An Itinerary is one complete way of getting from the start location to the end location.
+ * Pare down version of class original produced for OpenTripPlanner.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Itinerary {
+
+    /**
+     * Duration of the trip on this itinerary, in seconds.
+     */
+    public Long duration = 0L;
+
+    /**
+     * Time that the trip departs.
+     */
+    public Date startTime = null;
```

Comment:
Using `Date` for `startTime` and `endTime` is causing an issue when saving a trip under #21, I need to investigate more.

Reply:
For completeness, I have merge #25 into #21